### PR TITLE
mosquitto 1.4.14

### PR DESCRIPTION
**Pull request by ralight**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/mosquitto.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Mosquitto < Formula
   desc "Message broker implementing the MQTT protocol"
   homepage "https://mosquitto.org/"
-  url "https://mosquitto.org/files/source/mosquitto-1.4.12.tar.gz"
-  sha256 "b12e2353f92cb76882ce7b7bddfea022b461b38cbe6e7fb7c969d0daeb379cf7"
+  url "http://mosquitto.org/files/source/mosquitto-1.4.14.tar.gz"
```
Comment (JCount) [inline Formula/mosquitto.rb]: It looks like you accidentally switched from `https` to `http`. Could you switch it back? Thanks.

**Comment by ralight** [comment]
Sorry about that, it was my first time using bump-formula-pr.

**Comment by JCount** [comment]
No, problem. 😄

**Comment by JCount** [comment]
Thank you for your contribution to Homebrew, @ralight! 👍

**Pull request closed without merging**